projects/igniteui-angular/src/lib/services/overlay/overlay.spec.ts

```diff
@@ -3138,7 +3138,7 @@ describe('igxOverlay', () => {
             tick();
             wrapperElement = (fixture.nativeElement as HTMLElement)
                 .parentElement.getElementsByClassName(CLASS_OVERLAY_WRAPPER_MODAL)[0] as HTMLElement;
-            expect(wrapperElement.style.visibility).toEqual('hidden');
+            expect(wrapperElement).toBeUndefined();
```

Comment:
we are removing modal class on closing and should query wrapperElement with `igx-overlay__wrapper` class. Please change `CLASS_OVERLAY_WRAPPER_MODAL` with `CLASS_OVERLAY_WRAPPER` and check if visibility is hidden